widgets/hits.js

```diff
@@ -2,7 +2,12 @@ var React = require('react');
 
 var utils = require('../lib/utils.js');
 
-function hits({container = null, templates = {}, hitsPerPage = 20}) {
+function hits({
+    container = null,
+    templates = {},
+    transformData = {},
```

Comment:
Will be thought for users if they programmatically build transform values: they will have to widget.type === 'widget-with-transforms' ? 'transforms' : 'transform' which would be a pain